Update the READMEs for the Firebase Jobdispatcher binding

topgenorth opened the pull request:
Forgot to update the README for this, which provides a link to docs on Xamarin developer portal.

topgenorth committed:
Update the READMEs for the Firebase Jobdispatcher binding

dnfclas commented:
@topgenorth,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by .NET Foundation. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

xamarin-jenkins commented:
Hello! I'm the build bot for the Mono project. I need approval from a Mono team member to build this pull request. A team member should reply with "approve" to approve a build of this pull request, "whitelist" to whitelist this and all future pull requests from this contributor, or "build" to explicitly request a build, even if one has already been done. Contributors can ignore this message.

Redth commented:
whitelist

bholmes committed:
[Xamarin.Build.Download] Change iOS default to overwrite source dll

We have seen good results with Android already using this default.
Now we have noticed in iOS build logs that the source dll is showing up
in several places after it should have been changed.  It is possible that
we are (or will be soon) seeing a mix of dlls being sent to build tools.
See this bug report...

https://bugzilla.xamarin.com/show_bug.cgi?id=56498#c4

topgenorth committed:
Set the NuGet dependency for Xamarin.Android.Support.v4

topgenorth committed:
FJD is not a plugin.
